Week1/homework/1/createQueries.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const mysql = require('process-nextick-args/node_modules/mysql');
```

Comment:
any particular reason for using this version of mysql? :grin:
I guess it was autocomplete that gave this surprising require
```suggestion
const mysql = require('mysql');
```

Reply:
I remember what happened, I had my own at the top and then copied some code from right below that line downwards and I then got confused when I saw two `mysql` variables I just deleted mine and didn't notice it was the correct one haha